web/src/lib/components/user-settings-page/avatar-selector.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  import { mdiClose } from '@mdi/js';
+  import CircleIconButton from '../elements/buttons/circle-icon-button.svelte';
+  import { createEventDispatcher } from 'svelte';
+  import FullScreenModal from '../shared-components/full-screen-modal.svelte';
+  import UserAvatar from '../shared-components/user-avatar.svelte';
+  import { UpdateUserDtoAvatarColorEnum, UserResponseDto } from '@api';
+
+  export let user: UserResponseDto;
+
+  const dispatch = createEventDispatcher();
+  const colors: Array<UpdateUserDtoAvatarColorEnum> = [
```

Comment:
Can this just be Object.values(UpdateUserDtoAvatarColorEnum)? Otherwise you have to remember to update the list whenever it changes.